src/hooks/usePageViews.js

```diff
@@ -6,7 +6,12 @@ const isDev = process.env.NODE_ENV === 'development';
 
 const ANALYTICS_PROVIDERS = {
   debug: async (location) => (isDev && console.log(`Tracking navigation to ${location.pathname}`)),
-  googleAnalytics: async (location) => (window.ga && window.ga.send(['pageview', location.pathname])),
+  gtag: async (location) => {
+    return window.gtag && window.gtag('event', 'page_view', {
+      page_location: location.href,
+      page_path: location.pathname,
+    });
```

Comment:
What I understood from the [documentation](https://developers.google.com/analytics/devguides/collection/gtagjs/pages#manual_pageviews) is that the default behaviour would be to have the snippet on each page where a view needs to be tracked. 

To have manual events sent to the `/collect` endpoint, we have to deactivate the call made by the snippet (`gtag('config', '{{ analytics_tools_config.ga_code|escapejs }}', {send_page_view: false});`) and then add the code in this diff to manually track the page_view event.